src/eth1spec/spec.py

```diff
@@ -333,6 +348,143 @@ def signing_hash(tx: Transaction) -> Hash32:
     )
 
 
+def header_hash(header: Header) -> Hash32:
+    """
+    Computes the hash of a block Header.
```

Comment:
```suggestion
    Computes the hash of a block header.
```
I don't think this needs to be capitalized?